public/index.html

```diff
@@ -1,121 +1,247 @@
+
+  <div class="card">
+
+    <div class="survey">
+      <h1>Which of these technologies do you use?</h1>
+      <div class="option">
+        <div class="checkbox">
+          <svg xml:space="preserve" focusable="false" version="1.1" viewBox="0 0 24 24">
+            <path class="checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none"></path>
+          </svg>
+        </div>
+         Skaffold
+      </div>
+      <div class="option">
+        <div class="checkbox">
+          <svg xml:space="preserve" focusable="false" version="1.1" viewBox="0 0 24 24">
+            <path class="checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none"></path>
+          </svg>
+        </div>
+        Jib
+      </div>
+      <div class="option">
+        <div class="checkbox">
+          <svg xml:space="preserve" focusable="false" version="1.1" viewBox="0 0 24 24">
+            <path class="checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none"></path>
+          </svg>
+        </div>
+        Knative
+      </div>
+      <div class="option">
+        <div class="checkbox">
+          <svg xml:space="preserve" focusable="false" version="1.1" viewBox="0 0 24 24">
+            <path class="checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none"></path>
+          </svg>
+        </div>
+        App Engine
+      </div>
+      <div class="option nota">
+        <div class="checkbox">
+          <svg xml:space="preserve" focusable="false" version="1.1" viewBox="0 0 24 24">
+            <path class="checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none"></path>
+          </svg>
+        </div>
+        None of the above
+      </div>
+      <div class="submit-btn">Submit</div>
+    </div>
+
+    <h2 class="thanks">Thanks for voting! <br>Checkout the <a href="/results">results</a>.</h2>
+
+  </div>
+
 
-    document.querySelector('#submit').classList.add('loading');
-
-    fetch(url, {
-      method: 'POST',
-      body: JSON.stringify(votes),
-      headers: {
-        'Content-Type': 'application/json; charset=utf-8'
-      },
-    }).then((response) => {
-      document.querySelector('#submit').classList.remove('loading');
-      console.log(response);
-      if (response.ok) {
-        document.querySelector('#options').style.display = 'none';
-        document.querySelector('#middle').style.display = 'table';
+  <script>
+    const optionData = ['Skaffold', 'Jib', 'Knative', 'App Engine', 'None of the above'];
```

Comment:
We are repeating these names twice in this files. Can we just use this array to generate the html?
I am asking this because as part of the demo, I will have to update the questionnaire and doing it twice will be overwhelming.